packages/components/Checkbox/package.json

```diff
@@ -27,7 +27,8 @@
     "@fluentui-react-native/text": ">=0.9.1 <1.0.0",
     "@fluentui-react-native/tokens": ">=0.8.0 <1.0.0",
     "@uifabricshared/foundation-composable": ">=0.8.0 <1.0.0",
-    "@uifabricshared/foundation-settings": ">=0.8.0 <1.0.0"
+    "@uifabricshared/foundation-settings": ">=0.8.0 <1.0.0",
+    "@fluentui-react-native/icon": ">=0.5.1 <1.0.0"
```

Comment:
Could we keep this in alphabetical order? Doing `yarn add <package>` at the root of this specific package tends to auto-do that and I'd prefer to just adopt that for future adds to make diffs minimal and just have an accepted order.

Reply:
Sure. Thanks for mentioning this.